src/simulate.py

```diff
@@ -17,6 +17,7 @@ def simulate(num_time_steps,radius=[0.1],x=[0.5], y=[0.5], dx=[0.0], dy=[0.0], g
 
     simulation_output = []
     t = 0
+    bounces = 0
```

Comment:
distinguish between wall and ball bounces (it may be useful for the future and isn't hard to do)